Library/Formula/bork.rb

```diff
@@ -0,0 +1,14 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.tar.gz"
+  sha256 "3e8217d5fe4de94663dac74af505d05649c9d995998d19d4b247e5d3ecb1dd4b"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match /bork usage:/, shell_output("#{bin}/bork", 1)
```

Comment:
How do we pass the testpath variable now? Is there an equivalent formula I can peek into to adopt the changes into this one? I have no clue about Ruby or homebrew casks and it's features in general. I just want to be able to install bork via brew.